extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerProjectImporter.java

```diff
@@ -101,24 +101,6 @@ public boolean applies(IProgressMonitor monitor) throws OperationCanceledExcepti
             return false;
         }
 
-        for (java.nio.file.Path directory : directories) {
```

Comment:
Given that now the android support in gradle build server is not complete yet, I think we should keep the code here.

You can comment out it when build your private build. once the follow-ups are addressed, we can remove it.

Reply:
Addressed in https://github.com/microsoft/vscode-gradle/pull/1594/commits/059cc6dfbf388d3c991678797b90c4a5d03ff938